client/stylesheets/_toCat.scss

```diff
@@ -2202,18 +2252,23 @@ i:hover {
 
 .reportBtn {
     text-align: right;
-    margin-right: 3rem;
+    margin: 5px 0 100px 0;
 }
 
 .reportBtn button {
-    background-color: $color-spacered;
-    color: #fff;
-    font-size: 1.1rem;
-    width: 100px;
-    height: 50px;
-    margin: 0px;
-    outline: 0;
-    padding: 15px 20px;
+    padding: 15px 22px;
+    color: #fff ;
+    background-color: #f8504b;
+    border: 1px solid #f8504b;
+    box-shadow: 0 -1px 0 rgba(255, 255, 255, 0.1) inset;    
+    border-radius: 6px;
+    overflow: hidden;  
+    font-size: 1.2rem !important; 
+    float: right !important;
+}
+
+.reportBtn button:hover {
```

Comment:
Merge rule `.reportBtn button:hover` with rule on line 2253<br>Selector `reportBtn` should be written in lowercase with hyphens